README.md

```diff
@@ -622,12 +622,19 @@ cmd /c "SET AWS_ACCESS_KEY_ID=ASIAIOSFODNN7EXAMPLE & SET AWS_SECRET_ACCESS_KEY=w
 | `TEST_PASSWORD` | No | The database cluster password. | password |
 | `TEST_DB_USER` | No | User within the database that is identified with AWS IAM database authentication. This is used for AWS IAM authentication. | jane_doe |
 
+### Enable Logging
+Enable logging by specifying the `logger` parameter in the connection string. For more information on the configuration parameter, see [Debugging/Profiling](https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-connp-props-debugging-profiling.html).
+The driver currently does not support custom logging outside the usual logging frameworks like SLF4J. For more information on using SLF4J with the driver see [here](https://dev.mysql.com/doc/connector-j/5.1/en/connector-j-logging-slf4j.html).
+
 ## Known Issues
 ### SSLHandshakeException
 Using the driver with JDKs based on OpenJDK 8u292+ or OpenJDK 11.0.11+ will result in an exception: `SSLHandshakeException: No appropriate protocol`.
 This is due to OpenJDK disabling TLS 1.0 and 1.1 availability in `security.properties`. For additional information see "[Disable TLS 1.0 and TLS 1.1](https://java.com/en/configure_crypto.html#DisableTLS)".
 To resolve this exception, add the `enabledTLSProtocols=TLSv1.2` connection property when connecting to a database.
 
+### Read-Write Splitting
+The driver does not support read-write splitting yet. A possible solution for now is to utilize multiple connection pools and redirect read write traffic via the regular and read-only cluster endpoints.
```

Comment:
I'd rephrase it. I need to read it few times to understand.
"redirect read write traffic via the regular and read-only cluster endpoints" -> "send write traffic to a connection pool connected to writer cluster endpoint, and send read traffic to another pool connected to reader cluster endpoint"
?